internal/character/silverwolf/eidolon.go

```diff
@@ -24,6 +24,15 @@ func init() {
 			OnAdd: func(mod *modifier.ModifierInstance) {
 				mod.SetProperty(prop.EffectRES, -0.2)
 			},
+			OnBeforeDying: func(mod *modifier.ModifierInstance) {
+				if mod.Owner() == mod.Source() {
```

Comment:
this solution won't work because this modifier is attached to enemies. My recommendation for here is in the eidolon init, create a subscription on the `TargetDeath` event and then do loop to remove from all enemies when SW dies.